joomla/joomla-cms
# [5.2] replace scss variables by css variables

Pull Request for Issue #41105 .

### Summary of Changes

This PR will replace the SCSS variables used to adjust the styling of btn-secondary by css variables. 
With this change we are more in line with the rest of the css using css variables

### Testing Instructions

Before and after the test the look and feel of an element with className `btn-secondary` should be the same
- Have a Joomla website up and running
- Apply this PR
- Rebuild the assets using `npm run watch` (or similar)
- Refresh the page 
- Inspect the css inside Dev Toolbar

### Actual result BEFORE applying this Pull Request

![Scherm­afbeelding 2023-12-13 om 16 27 45](https://github.com/joomla/joomla-cms/assets/639822/66dee8cd-71ef-4d98-a9dc-697b54be6c63)

### Expected result AFTER applying this Pull Request

![Scherm­afbeelding 2023-12-13 om 16 28 22](https://github.com/joomla/joomla-cms/assets/639822/03c80896-d599-4780-865f-f9aaf2bf674e)


### Link to documentations
Please select:
